docs/url-file-uploads.md

```diff
@@ -0,0 +1,75 @@
+# URL File Uploads
+
+## Create a URL file upload
+
+To create a URL file upload, use the `createUrlFileUpload` [graphql multipart request](https://github.com/jaydenseric/graphql-multipart-request-spec).
```

Comment:
nit: We could link specifically to single file request as we do not support multifile or batching atm - https://github.com/jaydenseric/graphql-multipart-request-spec#single-file

Reply:
Also I think that `use the `createUrlFileUpload` mutation in [graphql multipart request]`.